gammapy/cube/sherpa_.py

```diff
@@ -1,13 +1,68 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
 import numpy as np
+from ..utils.energy import EnergyBounds
+import sherpa.astro.ui as sau
+from sherpa.astro.ui import erf
+import astropy.wcs as pywcs
 from sherpa.models import ArithmeticModel, Parameter, modelCacher1d
 from sherpa.data import DataND, BaseData
 from sherpa.utils.err import DataErr, NotImplementedErr
 from sherpa.utils import SherpaFloat, NoNewAttributesAfterInit, \
     print_fields, create_expr, calc_total_error, bool_cast, \
     filter_bins, interpolate, linear_interp
 
+"""
+Definition of the model NormGauss2DInt: Integrated 2D gaussian
+"""
+fwhm_to_sigma = 1 / (2 * np.sqrt(2 * np.log(2)))
+fwhm_to_sigma_erf = np.sqrt(2) * fwhm_to_sigma
+
+
+class NormGauss2DInt(ArithmeticModel):
+    """Integrated 2D gaussian for sherpa models
+    """
+    def __init__(self, name='normgauss2dint'):
+        # Gauss source parameters
+        self.wcs = pywcs.WCS()
+        self.coordsys = "galactic"  # default
+        self.binsize = 1.0
+        self.xpos = Parameter(name, 'xpos', 0)  # p[0]
+        self.ypos = Parameter(name, 'ypos', 0)  # p[1]
+        self.ampl = Parameter(name, 'ampl', 1)  # p[2]
+        self.fwhm = Parameter(name, 'fwhm', 1, min=0)  # p[3]
+        self.shape = None
+        self.n_ebins = None
+        ArithmeticModel.__init__(self, name, (self.xpos, self.ypos, self.ampl, self.fwhm))
+
+    def set_wcs(self, wcs):
```

Comment:
I don't see, that this method is being used somewhere (especially in calc()...), is that intended?